samples/matter/lock/README.rst

```diff
@@ -238,6 +238,29 @@ The PIN code is different depending on the build type:
 
 See `Testing door lock using Bluetooth LE with Nordic UART Service`_ for more information about how to test this feature.
 
+Factory data support
+====================
+
+.. matter_door_lock_sample_factory_data_start
+
+In this example factory data support is enabled by default for each build types except ``no_dfu``.
+This means that a new factory data set will be generated when building for the target board automatically.
+
+To disable factory data support set to ``n`` the following configs:
+
+   * :kconfig:option:`CONFIG_CHIP_FACTORY_DATA`
+   * :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_BUILD`
+
+For example, to build the example without factory data support for ``nrf52840dk_nrf52840`` target, run the following command:
+
+.. code-block:: console
+
+   west build -b nrf52840dk_nrf52840 -- -DCONFIG_CHIP_FACTORY_DATA=n -DCONFIG_CHIP_FACTORY_DATA_BUILD=n
+
+To learn more about factory data, read the :doc:`matter:nrfconnect_factory_data_configuration` page in the Matter documentation.
```

Comment:
```suggestion
In this sample, the factory data support is enabled by default for all build types except for the ``no_dfu`` build type.
This means that a new factory data set will be automatically generated when building for the target board.

To disable factory data support, set the following Kconfig options to ``n``:

   * :kconfig:option:`CONFIG_CHIP_FACTORY_DATA`
   * :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_BUILD`

To learn more about factory data, read the :doc:`matter:nrfconnect_factory_data_configuration` page in the Matter documentation.
```
I think the example should not be needed. How to build is explained in other places, and the Kconfigs can be set to ``n`` in different places.